src/components/@molecules/TransactionDialogManager/stage/TransactionStageModal.tsx

```diff
@@ -1,10 +1,11 @@
 import { queryOptions } from '@tanstack/react-query'
```

Comment:
<img width="651" alt="Screenshot 2025-01-23 at 12 42 19 PM" src="https://github.com/user-attachments/assets/3dfbadc0-1b8a-4d23-9cb5-476a4b87ff37" />
Perhaps "Open Wallet" is not the correct wording for a capsule transaction